packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -342,10 +342,14 @@ function build(
     makeVirtualObjectRepresentative,
     makeWeakStore,
     makeKind,
+    RepairedWeakMap,
+    RepairedWeakSet,
```

Comment:
subverted? tapped?

Reply:
I originally had "virtualized", which sent MarkM down a garden path, so he suggested "repaired" which I just grabbed.  Maybe "shadowed", "overridden"?  None of these terms imply the specific alterations that have been made, so none of them really stand out in my mind as preferable.  Don't know that it's a big deal since the name appears in only a tiny number of places.

Reply:
I don't understand "subverted" or "tapped". What would they mean in this context?

FWIW, I still like "repaired"

Reply:
The name is not a big deal and “repaired” is fine. Please stop reading 👉here👈.

Repaired mirrors the term we use in SES for similar things, which is nice. The downside is it implies that the basic WeakMap and WeakSet are “broken”. I suppose that complaint applies more generally and I don’t have a better name for SES repairs, which take otherwise non-broken things and make them _suitable_ for SES.

By “tapped” and “subverted” I was thinking in terms of your offline description, where it does the same job but sends information elsewhere, like a wire tap.

Reply:
Given the above discussion, I'm going to declare that we decided on "repaired".

Reply:
`VirtuallyAware`?